### DISCUSSION TITLE
Shders getting errors and couldn't render properly

### DISCUSSION THREAD

**Discussion by andysze79** [visualdesigncafe/nature-shaders#2] [Help]
The issue happens in Unity HDRP 2022.3.7f1 <DX12>. 
<img width="1592" alt="Screenshot 2024-01-19 115750" src="https://github.com/visualdesigncafe/nature-shaders/assets/44683224/d80db8fe-4f2a-4ef2-b852-34eaa8fd1b80">

**Comment by maxplooi** [comment]
Could you let me know what version of Nature Shaders you are using? The latest [Nature Shaders 2022](https://assetstore.unity.com/packages/vfx/shaders/nature-shaders-2022-266863) version should work fine with HDRP in Unity 2022.
